src/FormTaxonomies/Actions/EnqueueFormBuilderAssets.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Give\FormTaxonomies\Actions;
+
+use Give\FormTaxonomies\ViewModels\FormTaxonomyViewModel;
+use Give\Framework\Support\Facades\Scripts\ScriptAsset;
+
+/**
+ * @unreleased
+ */
+class EnqueueFormBuilderAssets
+{
+    /**
+     * @var FormTaxonomyViewModel
+     */
+    protected $viewModel;
+
+    /**
+     * @unreleased
+     */
+    public function __construct(FormTaxonomyViewModel $viewModel)
+    {
+        $this->viewModel = $viewModel;
+    }
+
+    /**
+     * @unreleased
+     */
+    public function __invoke()
+    {
+        if($this->viewModel->isFormTagsEnabled() || $this->viewModel->isFormCategoriesEnabled()) {
+
+            $scriptAsset = ScriptAsset::get(GIVE_PLUGIN_DIR . 'build/formTaxonomySettings.asset.php');
+
+            wp_enqueue_script(
+                'givewp-builder-taxonomy-settings',
+                GIVE_PLUGIN_URL . 'build/formTaxonomySettings.js',
+                $scriptAsset['dependencies'],
+                $scriptAsset['version'],
+                true
+            );
+
```

Comment:
Looks like we also need to enqueue the styles

```
wp_enqueue_style(
    'givewp-builder-taxonomy-settings',
    GIVE_PLUGIN_URL . 'build/style-formTaxonomySettings.css'
);
```
<img width="1294" alt="Screenshot 2024-08-16 at 8 20 26 AM" src="https://github.com/user-attachments/assets/d1507d1b-bb6d-494c-bcc8-af6f491807c5">

Reply:
Do the styles get stripped out and built separately? This was imported via JS, but if the build results in a separate asset then I'll enqueue that as well.

Reply:
Resolved, see [d845a9c](https://github.com/impress-org/givewp/pull/7463/commits/d845a9ca565bb4b2b79952bbc50185d3d083a765).